src/client/pages/Checkout/CheckoutPayment/PaymentDetails/PaymentDetails.tsx

```diff
@@ -5,10 +5,15 @@ import { useTextKeys } from 'utils/textKeys'
 import { Headline } from 'components/Headline/Headline'
 import {
   AdyenCheckout,
-  Wrapper,
+  Wrapper as AdyenWrapper,
 } from '../../../ConnectPayment/components/AdyenCheckout'
 const { gray100, gray600, gray700, gray300, gray900 } = colorsV3
 
+const Wrapper = styled.div`
+  margin: 0 auto;
+  max-width: 628px;
```

Comment:
Do we have a constant for this? Maybe we should use rems at least?

Reply:
rems is a bit strange for this. it would be 39.25. 

I will add a const WrapperWidth and we can use it in all files part of the Checkout